pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -90,8 +90,10 @@ def optimizer_step(
         # in manual optimization, the closure does not return a value
```

Comment:
If I understand it correctly, this comment is no longer true.
```suggestion
```

Reply:
It is true. Why do you think it isnt?